include/onnxruntime/core/framework/kernel_registry.h

```diff
@@ -6,6 +6,9 @@
 #include "core/framework/op_kernel.h"
 
 namespace onnxruntime {
+
+using KernelCreateMap = std::multimap<std::string, KernelCreateInfo>;
+
```

Comment:
This could be scoped lower as I think the only usage is for the private kernel_creator_fn_map_ member